src/components/molecules/Accordion/AccordionService.ts

```diff
@@ -0,0 +1,19 @@
+export class AccordionService {
```

Comment:
The purpose of this class is to coordinate state between all of the accordion items. So when an item is registered it will put a reference to it's reactive state in here, and then they can signify that one is opening and the others should close. We can utilize this to add more options like only allowing the next one to be open etc.